docs/triply-api/index.md

```diff
@@ -88,6 +88,29 @@ the Pokémon dataset:
 https://api.triplydb.com/datasets/academy/pokemon/
 ```
 
+### LD Browser API
+
+Triply APIs provide a convenient way to access data used by [LD Browser](https://triply.cc/docs/triply-db-getting-started/#linked-data-browser), which offers a comprehensive overview of a specific IRI. By using Triply API for a specific IRI, you can retrieve the associated 'document' in the .nt format that describes the IRI.
+
+To make an API request for a specific instance, you can use the following URI path:
+
+```none
+https://api.triplydb.com/datasets/ACCOUNT/DATASET/browser?-resource=RESOURCE
+```
+
+To illustrate this, let's take the example of the DBpedia dataset and the [specific instance of 'Mona Lisa'](https://triplydb.com/DBpedia-association/dbpedia/browser?resource=http%3A%2F%2Fdbpedia.org%2Fresource%2FMona_Lisa). If you use this URI path: 
+
+```none
+https://triplydb.com/DBpedia-association/dbpedia/browser?resource=http%3A%2F%2Fdbpedia.org%2Fresource%2FMona_Lisa
```

Comment:
This is not a correct URI path, it's a link to Mona Lisa in DBpedia LD Browser, it doesn't produce a .nt document